### ISSUE TITLE
Another `antfu/consistent-list-newline` false positive with parentheses

### ISSUE THREAD

**Issue by sirxemic** [antfu/eslint-plugin-antfu#27]
### Describe the bug

This seems similar to https://github.com/antfu/eslint-plugin-antfu/issues/19, the only difference being it's a function call this time instead of an array.

Take the following example:

```js
this.foobar(
  (x),
  y,
  z
)
```

This gets fixed to:
```js
this.foobar(
  (x), y, z)
```

I would expect it to just keep it as-is.

In my specific case I am actually doing `(1 << x)`. WebStorm or something really seems to not like the code if I don't use parentheses.

### Reproduction

See above.

### System Info

```Shell
N/A
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

### Contributions

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests (actually just go ahead and do it, thanks!)
